src/app/client/src/app/modules/shared/components/card/card.component.html

```diff
@@ -3,7 +3,7 @@
     <div class="top attached indicating active " *ngIf="data.maxCount && data.progress > 0">
       <sui-progress class="top attached active" [value]="data.progress" [maximum]="data.maxCount"></sui-progress>
     </div>
-    <div class="ui image" (click)="onAction(data,data.action.onImage)">
+    <div class="ui image" (click)="onAction(data,data.action.onImage)"  appTelemetryInteract [telemetryInteractObject]="{id:data.metaData.identifier,type:data.telemetryObjectType,ver:'1.0'}" [telemetryInteractEdata]="data.telemetryInteractEdata">
```

Comment:
version number is dynamic and it is based on type for content it contains content version